packages/main/src/browser.ts

```diff
@@ -3,13 +3,16 @@ import { isMainError } from './error';
 
 export * from './error';
 
+const DEFAULT_NAME = 'entry point';
+
```

Comment:
I don't think having a constant here adds any clarity

Reply:
I extract it to the constant, because value is used twice.